resources/views/vendor/mail/html/themes/default.css

```diff
@@ -270,6 +270,7 @@ img {
     background-color: #edf2f7;
     color: #718096;
     padding: 16px;
+    word-break: break-all;
```

Comment:
For some reason, the original test case from [here](https://retroachievements.org/viewtopic.php?t=29236&c=275728#275728) has some correct newlines on master, and some double `<br>`s:
![image](https://github.com/user-attachments/assets/e2b84fdc-9426-492b-9cbe-e444bffa1f77)

Whereas, everything is single `<br/>`s (but properly newlined) when submitted on this branch:
![image](https://github.com/user-attachments/assets/b23c43c1-919c-4c47-a71a-993e0acb6116)

Reply:
But the wrapping does seem to be fixed.
![image](https://github.com/user-attachments/assets/c3d44a34-c8d5-4c28-b0d7-2fe1cee50729)

Presumably this will prevent Outlook from trying to make the whole thing visible by shrinking it.

Reply:
Think I have both line break cases fixed in latest:

![Screenshot 2025-03-28 at 6 47 29 PM](https://github.com/user-attachments/assets/5cda2ec0-9f82-44f8-8a9f-0a26946e9a8f)

![Screenshot 2025-03-28 at 6 47 36 PM](https://github.com/user-attachments/assets/ec0cb2b5-a90f-4213-930d-8d2d800fe5ff)